x-pack/plugins/cases/public/components/all_cases/all_cases_list.test.tsx

```diff
@@ -295,22 +300,28 @@ describe('AllCasesListGeneric', () => {
     expect(screen.getByTestId('tableHeaderCell_title_0')).toBeInTheDocument();
   });
 
+  it('renders the category column', async () => {
+    appMockRenderer.render(<AllCasesList />);
+
+    expect(screen.getByTestId('tableHeaderCell_category_4')).toBeInTheDocument();
```

Comment:
nit: I know we have the same pattern but does `getByText('Categoyr')` work? If feels more natural.